index.html

```diff
@@ -1,25 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Awesome Books Project</title>
-  <link rel="stylesheet" href="style.css"> 
+  <link rel="stylesheet" href="style.css">
   <script src="script.js" defer></script>
 </head>
+
 <body>
-  <h1 class="header">All Awesome books</h1>
-  <div class="booksContainer">
+  <header>
+    <h1>Awesome Books</h1>
+    <div>
+      <ul id="nav-links">
+        <li> <a href="#" class="links book-list-link" id="defaultOpen">List</a> </li>
+        <li> <a href="#" class="links add-new-book">Add new</a> </li>
+        <li> <a href="#" class="links contact-sec">Contact</a> </li>
+      </ul>
+    </div>
+  </header>
```

Comment:
- [ ] You currently have not implemented the date and time right below your navigation bar as shown in the wireframe below 👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/170798709-0ce69cd1-1d3b-4cfe-b878-4a4039ea096e.png)
### Wireframe Guideline
![image](https://user-images.githubusercontent.com/46839548/170798766-0aed35d9-56e2-4244-9fc7-58058f57e4b9.png)

You can use `luxon.js` to achieve this easily 👍